src/status_im2/subs/wallet/networks.cljs

```diff
@@ -0,0 +1,42 @@
+(ns status-im2.subs.wallet.networks
+  (:require [quo.foundations.resources :as resources]
+            [re-frame.core :as re-frame]
+            [status-im2.constants :as constants]))
+
+(re-frame/reg-sub
+ :wallet/filtered-networks-by-mode
+ :<- [:wallet/networks]
+ (fn [networks [_ test?]]
+   (get networks (if test? :test :prod))))
+
+(def ^:const mainnet-network-details
+  {:source       (resources/networks :ethereum)
+   :short-name   "eth"
+   :network-name :ethereum})
+
+(def ^:const arbitrum-network-details
+  {:source       (resources/networks :arbitrum)
+   :short-name   "arb1"
+   :network-name :arbitrum})
+
+(def ^:const optimism-network-details
+  {:source       (resources/networks :optimism)
+   :short-name   "opt"
+   :network-name :optimism})
+
+(def ^:const network-list
```

Comment:
We just need to be mindful `^:const` will inline everything on every usage. For values that are not simple numbers or strings we need to be careful with memory consumption. I would personally not use `^:const` for these vars.

Reply:
ok sure, I will remove that 👍